### DISCUSSION TITLE
Incorporating Cooling Energy in Calculations

### DISCUSSION THREAD

**Discussion by vchemla** [mlco2/ecologits#107] [Q&A] [answered]
Hi all,

I am wondering if your project has taken into account the cooling energy consumption.

In fact, the energy used for fans or water cooling should also be considered for GPUs and CPUs.

Thank you in advance

**Comment by adrienbanse** [comment, marked as answer]
Hey @vchemla, 

It's included in the PUE: ["_We [...] also take into account the energy consumption of cooling equipment integrated with the data center, using the Power Usage Effectiveness (PUE) metric._"](https://ecologits.ai/latest/methodology/llm_inference/)
